pkgs/by-name/py/pyroveil/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  python3,
+  makeWrapper,
+}: stdenv.mkDerivation (final: {
+  pname = "pyroveil";
+  version = "e7d5a7fe3bfd6313509ad6d4cc5d7685c7b2098a";
+
+  src = fetchFromGitHub {
+    owner = "HansKristian-Work";
+    repo = "pyroveil";
+    rev = final.version;
+    fetchSubmodules = true;
+    hash = "sha256-93GwfnyUJBfWfGDidiPHy+sKQGMToAdNOim4NDV9odg=";
+  };
+
+  patches = [
+    ./0001-cmake-fix-usage-of-CMAKE_INSTALL_LIBDIR.patch
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    python3
+    makeWrapper
+  ];
+
+  init = ./init.sh;
+
+  postInstall = ''
+    mkdir -p "$out/share/pyroveil"
+    cp -r ../hacks "$out/share/pyroveil"
+    install -D "$init" "$out/bin/pyroveil-init"
```

Comment:
```suggestion
    install -D "${./init.sh}" "$out/bin/pyroveil-init"
```

Reply:
Do you have any link, why I should not use env in such cases?